renovate.json5

```diff
@@ -32,7 +32,7 @@
 
     assigneesFromCodeOwners: true,
 
-    ignorePaths: ['**/templates/**', '**/test-input/**', '**/test-output/**', '**/test/fixtures/**', '**/test/test-data/**'],
+    ignorePaths: ['**/templates/**', '**/test-input/**', '**/test-output/**', '**/test/fixtures/**', '**/test/test-data/**', '**/fixture/**', '**/fixtures/**'],
```

Comment:
Does it make sense to add also `**/test/fixture/**`?

Reply:
Done in https://github.com/SAP/open-ux-tools/pull/1240/commits/246d2feba47a11ee128c0ce856fd99ed7b870b25 Thanks